frontend/src/metabase/dashboard/components/DashboardHistoryModal.jsx

```diff
@@ -4,24 +4,37 @@ import { withRouter } from "react-router";
 
 import HistoryModal from "metabase/containers/HistoryModal";
 import * as Urls from "metabase/lib/urls";
+import * as dashboardActions from "metabase/dashboard/actions";
 import Dashboards from "metabase/entities/dashboards";
+import connect from "react-redux/lib/connect/connect";
```

Comment:
1. Can be `import { connect } from "react-redux"`
2. Could you please move the `connect` import to the top? Under `react` maybe. See the [FE style guide](https://github.com/metabase/metabase/blob/master/docs/developers-guide/frontend.md#javascript-conventions) for details

Reply:
Sure!